docs/manual/workflow-steps/datadog.md

```diff
@@ -60,3 +60,95 @@ Where `path` is the path created above.
 
 - **Tags**
 : A list of tags that you want to be applied to the event. To specify more than one separate each by a comma.
+
+:::warning
+The following job steps require specifying the API and APP key paths in either project settings (e.g. project.datadog.api_key_path=keys/datadog/api & project.datadog.app_key_path=keys/datadog/app) or framework.properties (e.g. datadog.api_key_path=keys/datadog/api & datadog.app_key_path=keys/datadog/app) before running the step as it need to pull severity/state values specific to the account.
+:::
+
+## Datadog / Incidents / Create Incident
+
+![Datadog - Create Incident](~@assets/img/dd-create-incident.png)
+
+- **Title**: The title of the new event being created.
+
+- **Incident Commander**: The ID of the user who should be assigned as the commander on the incident.
```

Comment:
Is it obvious where they would find this ID?  Do we need to give them a hint or instructions?